runtime/compiler/control/MethodToBeCompiled.cpp

```diff
@@ -96,7 +96,7 @@ void TR_MethodToBeCompiled::initialize(TR::IlGeneratorMethodDetails & details, v
 void
 TR_MethodToBeCompiled::shutdown()
    {
-   cleanupJITaaS();
+   cleanupJITServer();
```

Comment:
In another PR the name of this method has been changed

Reply:
#6893 changes `cleanupJITaaS` to `freeJITServerAllocations`. I assume this PR will be merged first. Then I'll do rebase for PR #6893. This PR is dedicated to replace all `jitaas` names.